server/vulnerabilities/cpe.go

```diff
@@ -151,21 +153,21 @@ func CPEFromSoftware(db *sqlx.DB, software *fleet.Software) (string, error) {
 	checkTargetSW := ""
 	args := []interface{}{cleanAppName(software.Name)}
 	if targetSW != "" {
-		checkTargetSW = " AND target_sw MATCH ?"
+		checkTargetSW = " AND target_sw=?"
```

Comment:
This change is because with MATCH some values are interpreted weirdly, like if they have a . or a - sqlite tries to match a column instead of considering a string.